sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -186,6 +187,11 @@ export interface StoragePipelineOptions {
    * Configures the HTTP client to send requests and receive responses.
    */
   httpClient?: IHttpClient;
+
+  /// <summary>
+  /// The audience used to retrieve an AAD token.
+  /// </summary>
+  Audience?: BlobAudience;
```

Comment:
Should start with lower case `audience`.  Also could we make it a `string` and use the two constants instead?

Reply:
Thanks a lot for the reviewing. 

I was trying to remind customer that currently only storage and disk audience would will be available here.  With a plain string type, what would you suggest to tell customer this information?

Reply:
In general, we want to avoid using classes. We also already have the prior art of using `string` type for `audience` in the SDK.   One option is to export a constant that has the two audience:

```ts
export const KnownStorageAudiences = {
  StorageAudience: "...",
  DiskComputeOAuthScopes: "...",
}
```
and mention them in the ref doc of `audience` option.